src/api/ApiClient.js

```diff
@@ -3,16 +3,44 @@ import base64 from 'base-64';
 
 const apiVersion = 'api/v1';
 
+type Account = {
```

Comment:
I am adding export to all types, but let's keep this there for now.
The reducers may have different representation of the same data...
In fact it is the action creators that need the type info, while the reducers need to be more like a black box, so you can change them freely without breaking code.